draft-ietf-ppm-dap.md

```diff
@@ -1395,8 +1419,14 @@ struct {
   HpkeCiphertext leader_encrypted_input_share;
   HpkeCiphertext helper_encrypted_input_share;
 } Report;
+
+struct {
+  Report reports[report_count];
```

Comment:
`report_count` is misleading. Per RFC 8446 vector syntax ([1](https://datatracker.ietf.org/doc/html/rfc8446#section-3.4)), the value in square brackets is the length *in bytes*, not the count of elements in the vector. Let's use `content_length` as in https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/commit/4867378c805dfb961aacd4be0c975d26946c1922.